### PR TITLE
Feat: add feature to move bubble arrow and block window scroll

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
- Now we can set bubble arrow position as per requirement ['top', 'right', 'bottom', 'left']
- we can enable/disable window scrolling for tour

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -299,8 +306,9 @@ steps: PropTypes.arrayOf(
               ]).isRequired,
             position: PropTypes.oneOfType([
                 PropTypes.arrayOf(PropTypes.number),
-                PropTypes.oneOf(['top', 'right', 'bottom', 'left', 'center']),
+                PropTypes.oneOf(['top', 'right', 'bottom', 'left', 'centre']),
```
Comment (aashitk) [inline README.md:309]: Typo here, it should be `center`

File: dist/reacttutorial.min.js [hunk 2]
```diff
@@ -3,4 +3,4 @@
...
...!z&&ie===c.length-1,inverted:!0,label:z&&ie===c.length-1?z:H||null}))),o.a.createElement(F,{windowWidth:xe.w,windowHeight:xe.h,targetWidth:xe.width,targetHeight:xe.height,targetTop:xe.top,targetLeft:xe.left,padding:j,rounded:P,className:R,disableInteraction:!1===c[ie].stepInteraction||T?!c[ie].stepInteraction:T,maskColor:K,disableInteractionClassName:s()(Q.disableInteraction,C)})):null},se={top:0,right:0,bottom:0,left:0,width:0,height:0,w:0,h:0,node:null};function de(e,t){switch(t.type){case"HAS_DOM_NODE":t.type;var n=te(t,ee);return re(re({},e),n);case"NO_DOM_NODE":return re(re({},e),{},{top:e.h+10,right:e.w/2+9,bottom:e.h/2+9,left:t.w/2-e.helperWidth?e.helperWidth/2:0,width:0,height:0,w:t.w,h:t.h,helperPosition:"center",node:null});default:return e}}var fe={CLICK:"click",DBL_CLICK:"dblclick",TYPING:"typing",ENTER:"enter",DRAG_N_DROP:"dragdrop",DRAG_WITH_MOUSE_MOVE:"dragwithmove",CUSTOM:"custom",WAIT:"wait"};ue.propTypes=$,ue.defaultProps={showNavigation:!0,showNavigationNumber:!0,showButtons:!0,showNumber:!0,startAt:0,scrollDuration:1,maskSpace:10,stepWaitTimer:0,disableInteraction:!1,rounded:0,accentColor:"#007aff",closeWithMask:!0,maskColor:"#000"};var pe=ue;t.default=pe}])}));
```
Comment (aashitk) [inline dist/reacttutorial.min.js:6]: Let 's not commit minified files with changes
We always have separate commit of bundled min files with version update

**Review by aashitk — CHANGES_REQUESTED** [review]
Few comments added

File: demo/App.js [hunk 3]
```diff
@@ -26,14 +26,16 @@ function App() {
     {
         content: 'double click this item',
         actionType: 'dblclick',
-        position: 'top',
+        position: 'right',
+        arrowPosition: 'left',
```
Comment (aashitk) [inline demo/App.js:30]: Can't arrow position be anticipated from `position` prop?

**Review by aashitk — CHANGES_REQUESTED** [review]
One minor comment

File: demo/App.js (hunk 3 above)
Comment (ShaileshParmar11) [reply demo/App.js:30]: position can contain either string or array of number so we need extra variable to get arrow position
New

**Review by aashitk — APPROVED** [review]
LGTM!

**Pull request merged**